core/block.js

```diff
@@ -985,6 +985,11 @@ Blockly.Block.prototype.appendDummyInput = function(opt_name) {
  * @param {!Object} json Structured data describing the block.
  */
 Blockly.Block.prototype.jsonInit = function(json) {
+  // Add the mutator to the block
+  if (json['mutator'] !== undefined) {
+    Blockly.Extensions.apply(json['mutator'], this);
+  }
```

Comment:
Check domToMutation/mutationtoDom exist after apply. The extension may not be defined by registerMutator().

Also check those didn't change after each json.extensions entry (line 1051).